ztp/source-crs/PtpConfigGmWpc.yaml

```diff
@@ -0,0 +1,150 @@
+# The grandmaster profile is provided for testing only
+# It is not installed on production clusters
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: grandmaster
+  namespace: openshift-ptp
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  profile:
+  - name: "grandmaster"
+    ptp4lOpts: "-2 --summary_interval -4"
+    phc2sysOpts: -r -u 0 -m -O -37 -N 8 -R 16 -s $iface_master -n 24
+    ptpSchedulingPolicy: SCHED_FIFO
+    ptpSchedulingPriority: 10
+    plugins:
+      e810:
+        enableDefaultConfig: true
+    ts2phcOpts: " "
+    ts2phcConf: |
+      [nmea]
+      ts2phc.master 1
+      [global]
+      use_syslog  0
+      verbose 1
+      logging_level 7
+      ts2phc.pulsewidth 100000000
+      #GNSS module s /dev/ttyGNSS* -al use _0
+      #cat /dev/ttyGNSS_1700_0 to find available serial port
+      #example value of gnss_serialport is /dev/ttyGNSS_1700_0
+      ts2phc.nmea_serialport $gnss_serialport
+      leapfile  /usr/share/zoneinfo/leap-seconds.list
+      [$iface_master]
+      ts2phc.extts_polarity rising
+      ts2phc.extts_correction 0
+    ptp4lConf: |
+      [$iface_master]
+      masterOnly 1
+      [$iface_master_1]
+      masterOnly 1
+      [$iface_master_2]
+      masterOnly 1
+      [$iface_master_3]
+      masterOnly 1
+      [global]
+      #
+      # Default Data Set
+      #
+      twoStepFlag 1
+      priority1 128
+      priority2 128
+      domainNumber 24
+      #utc_offset 37
+      clock_class_threshold 165
+      clockClass 248
+      clockAccuracy 0xFE
```

Comment:
For E810, I believe this is within 50 μs, which would gfall in this category
0x27:  The time is accurate to within 100 μs

Reply:
done